src/constructs/cloudwatch/ec2-alarms.test.ts

```diff
@@ -65,9 +66,25 @@ describe("The Gu5xxPercentageAlarm construct", () => {
       numberOfMinutesAboveThresholdBeforeAlarm: 3,
       snsTopicName: "test-topic",
     };
-    new Gu5xxPercentageAlarm(stack, "test", { ...app, loadBalancer: alb, ...props });
+    new Gu5xxPercentageAlarm(stack, { ...app, loadBalancer: alb, ...props });
     expect(stack).toHaveResource("AWS::CloudWatch::Alarm", {
       EvaluationPeriods: 3,
     });
   });
 });
+
+describe("The GuUnhealthyHostsAlarm construct", () => {
+  it("should create the correct alarm resource with minimal config", () => {
+    const stack = simpleGuStackForTesting();
+    const alb = new GuApplicationLoadBalancer(stack, "ApplicationLoadBalancer", { ...app, vpc });
```

Comment:
Unfortunately you need all 3 ALB components wired up in order to access the `metricUnhealthyHostCount()`metric!